.gitignore

```diff
@@ -21,4 +21,4 @@ site
 .vscode/
 esdoc
 node_modules
-lib
+#lib
```

Comment:
Why this change in `.gitignore`, you should only work on `src` folder

Reply:
My bad, I was testing it. Updating the PR with the change

Reply:
@sushantdhiman I have updated the PR